Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -748,107 +658,107 @@ protected virtual IHttpClient CreateHttpClient()
         /// <summary>
         /// Registers resources that classes will depend on
         /// </summary>
-        protected void RegisterResources()
+        protected void RegisterResources(IServiceCollection serviceCollection)
         {
-            RegisterSingleInstance(ConfigurationManager);
-            RegisterSingleInstance<IApplicationHost>(this);
+            serviceCollection.AddSingleton(ConfigurationManager);
+            serviceCollection.AddSingleton<IApplicationHost>(this);
 
-            RegisterSingleInstance<IApplicationPaths>(ApplicationPaths);
+            serviceCollection.AddSingleton<IApplicationPaths>(ApplicationPaths);
 
-            RegisterSingleInstance(JsonSerializer);
 
-            RegisterSingleInstance(LoggerFactory, false);
-            RegisterSingleInstance(Logger);
+            serviceCollection.AddSingleton(JsonSerializer);
 
-            RegisterSingleInstance(EnvironmentInfo);
+            serviceCollection.AddSingleton(LoggerFactory);
+            serviceCollection.AddLogging();
```

Comment:
Does this registration add anything we need specifically? we've already registered the factory and the logger.

Reply:
Yes it adds support for ‘Logger<T>’